File: dayThree/Javascript/ReverseString.js
let text = "Hello world";
        
        let reverse = "";
        
        for (let count = text.length-1; count >= 0; count--){
        
        let alphabet = text.charAt(count).toLowerCase();
        
               reverse += alphabet;
         
        }
     
console.log(reverse);
     
     

File: dayThree/java/ReverseString.java
public class ReverseString {

    public static void main(String[] args){
    
        String text = "Hello world";
        
        String reverse = "";
        
        for (int count = text.length()-1; count >= 0; count--){
        
        char alphabet = Character.toLowerCase(text.charAt(count));
        
               reverse += alphabet;
         
        }
     
     System.out.println(reverse);   
    }       
    
    
}     
